ARSIS-Rover/rover.env

```diff
@@ -0,0 +1,2 @@
+TELEMETRY_API=172.17.0.1:8080
```

Comment:
How should we handle this for deployment/testing?

Reply:
We could probably use local DNS using the access point DHCP configuration

Reply:
This is so when we deploy it to a pi we have an easy way of setting where to look for the api